samples/tests/create_batch_prediction_job_video_classification_test.py

```diff
@@ -0,0 +1,75 @@
+# Generated code sample for google.cloud.aiplatform.PipelineServiceClient.create_training_pipeline
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from uuid import uuid4
+import pytest
+import os
+
+from samples import (
+    create_batch_prediction_job_video_classification_sample,
+    cancel_batch_prediction_job_sample,
+    delete_batch_prediction_job_sample,
+)
+
+PROJECT_ID = os.getenv("BUILD_SPECIFIC_GCLOUD_PROJECT")
+LOCATION = "us-central1"
+MODEL_ID = "667940119734386688"  # Permanent 5 class sports model
+DISPLAY_NAME = f"temp_create_batch_prediction_vcn_test_{uuid4()}"
+GCS_SOURCE_URI = (
+    "gs://ucaip-samples-test-output/inputs/vcn_40_batch_prediction_input.jsonl"
+)
+GCS_OUTPUT_URI = "gs://ucaip-samples-test-output/"
+
+
+@pytest.fixture
+def shared_state():
```

Comment:
Why do you have two separate fixture to achieve this? I'd only have one fixture. You'll encounter weird situation when those fixtures have different scope.

Maybe:

```python
@pytest.fixture(scope="function")
def deleter()
    state = {}
    yield state
    # any cleanup tasks

def test_something(capsys, deleter):
    # create something
    deleter["batch_prediction_job_name"] = out.split("name:")[1].split("\n")[0]

```